service/storage_service.cc

```diff
@@ -1732,10 +1735,8 @@ storage_service::prepare_replacement_info(std::unordered_set<gms::inet_address>
     auto dc_rack = get_dc_rack_for(replace_address);
     auto raft_id = get_raft_id_for(_gossiper, replace_address);
 
-    // use the replacee's host Id as our own so we receive hints, etc
-    auto host_id = _gossiper.get_host_id(replace_address);
-    co_await _sys_ks.local().set_local_host_id(host_id).discard_result();
-    const_cast<db::config&>(_db.local().get_config()).host_id = host_id; // FIXME -- carry non-cost config on storage service itself
+    auto replace_host_id = _gossiper.get_host_id(replace_address);
+    slogger.info("Host {}/{} is replacing {}/{} using a different address", _db.local().get_config().host_id, get_broadcast_address(), replace_host_id, replace_address);
```

Comment:
`prepare_replacement_info` is called in both cases: replacing with the same and with a different address.
So this will also be printed when replacing with the same address.

I already queued, we can fix this in a follow-up.